package.json

```diff
@@ -13,10 +13,15 @@
     "@next/font": "^14.0.2",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "local-storage": "^2.0.0",
+    "local-storage-data": "^1.0.5",
     "next": "^13.5.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.2"
+    "styled-components": "^6.0.2",
+    "uid": "^2.0.2",
+    "use-local-storage": "^3.0.0",
+    "use-local-storage-state": "^19.1.0"
```

Comment:
I think you are only using `use-local-storage-state`, which means you can and should remove all the libraries for local-storage you don't need.

To uninstall a library, you can type the command `npm uninstall <nameOfTheLibrary>`.
You should do this for the libraries "local-storage", "local-storage-data" and "use-local-storage".